DragaliaAPI/Controllers/DragaliaControllerBase.cs

```diff
@@ -21,7 +21,17 @@ public override OkObjectResult Ok(object? value)
         return base.Ok(
             new DragaliaResponse<object>(
                 value ?? throw new ArgumentNullException(nameof(value)),
-                ResultCode.Success
+                ResultCode.SUCCESS
+            )
+        );
+    }
+
+    public OkObjectResult Ok(ResultCode resultCode, object? value)
+    {
+        return base.Ok(
+            new DragaliaResponse<object>(
+                value ?? throw new ArgumentNullException(nameof(value)),
+                resultCode
             )
         );
     }
```

Comment:
as mentioned before this is now achieved via DragaliaExceptions, so this overload shouldn't exist -- idk if you added it